### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by iw0nderhow**
Not much to say here, I think.

### CODE REVIEW

**Comment by PowerLemons** [comment]
May I ask where you got this translation from :)?

**Comment by iw0nderhow** [comment]
I did it myself, is there anything wrong with it :)?

**Comment by Kirvx** [comment]
@karottenbaum You can delete dgc-browser-modal-moolah and all dgc-getting-started-web-moolah since Moolah is deleted https://github.com/dogecoin/dogecoin.com/pull/57

**Comment by iw0nderhow** [comment]
@Kirvx Is it correct that the descriptor (or what it is called) on line 224 mentions Block.io but the translation still mentions DogeAPI or should the translation be changed to mention Block.io?

**Comment by PowerLemons** [comment]
I don't speak German, so I don't know whether or not this translation is accurate. Could you get other German shibes to vouch for your translation?

**Comment by iw0nderhow** [comment]
I will post this in the German subreddit, but I have one question: How do I present a working page to people? On my github.io page, the images won't load. Can I use my gh-pages branch, which I used for the pull request, or must I duplicate the repository and change things there?

**Comment by Kirvx** [comment]
DogeAPI will be replaced to Block.io (https://github.com/dogecoin/dogecoin.com/pull/53), so descriptors are ok.

**Comment by PowerLemons** [comment]
@karottenbaum I don't think a fully-working page is needed. Simply hand them a copy of the http://dogecoin.com page and your broken page and they'll know what it's suppose to look like.

**Comment by iw0nderhow** [comment]
Well, my page does not work at all. In this repository, images and scripts
won't load and in my test repository, images and scripts load, but fail to
read the locale strings.

**Comment by dhpollack** [comment]
@karottenbaum @PowerLemons @Kirvx 

http://dhpollack.github.io/?lng=de

You can use that address to test this pull request.  There are two errors that I had to fix.

File: i18n/de.json [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+    "dgc-wallet-source": " Quellcode",
+    "dgc-scroll-button-link": "Klicke, um nach unten zu scrollen",
+    "dgc-scroll-button-alt": "Nach-unten-Scroll-Knopf",
+    "dgc-dogepng-alt": "Doge",
+    "dgc-what-is-doge": "Was ist Dogecoin?",
+    "dgc-quick-desc": "Dogecoin ist eine dezentralisierte Peer-zu-Peer-Digitalw&auml;hrung, welche dir erm&ouml;glicht, bequem online Geld zu versenden. Denke von Dogecoin als \"die Internetw&auml;hrung.\"",
+    "dgc-get-started-now": "Erste Schritte",
+    "dgc-learn-more": "Mehr erfahren",
+    "dgc-what-with-dog": "Was ist mit Dogecoin und dem Hund?",
+    "dgc-doge-explained": "\"Doge\“ ist unser spa&szlig;iges, freundliches Maskottchen! Der Shiba Inu ist eine japanische Hunderasse, welche als ein Meme online popul&auml;r geworden ist und Dogecoin repr&auml;sentiert.",
```
Comment (dhpollack) [inline i18n/de.json]: The second " is not correct.  I believe it's a character that's on european keyboards and not US keyboards.  You need to change this to the correct parathesis :)

File: i18n/de.json [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    "dgc-browser-modal-dogechain": "DogeChain Wallet",
+    "dgc-browser-modal-holytransaction": "HolyTransaction",
+    "dgc-browser-modal-dogeapi": "DogeAPI",
+    "dgc-browser-modal-blockio": "Block.io",
+    "dgc-browser-modal-online-disclaimer": "<strong>NOTIZ:</strong> Dogecoin oder die Dogecoin-Stiftung besitzen und betreiben diese Online-Brieftaschen nicht. Wir denken zwar, dass diese Unternehmen zuverl&auml;ssig sind, allerdings benutzt Du diese Online-Brieftaschen <strong>auf eigene Gefahr</strong>.",
+    "dgc-ios-wallet-modal-important": "WICHTIG:",
+    "dgc-ios-wallet-modal-view-only": "MyDoge f&uuml;r iOS kann nur benutzt werden, um Brieftaschen anzusehen.",
+    "dgc-ios-wallet-modal-not-send": "MyDoge <strong>kann keine Dogecoins senden.</strong> Du musst erst eine tats&auml;chliche Brieftasche einrichten, bevor du dieses Programm zum Ansehen derselben benutzt.",
+    "dgc-ios-wallet-modal-understand": "Ich habe verstanden, los geht\'s!",
```
Comment (dhpollack) [inline i18n/de.json]: The "\" should not be here because single quotes do not need to be escaped within double quotes.

**Comment by dhpollack** [comment]
@karottenbaum One more thing.  The getting-started text doesn't translate again because of a European keyboard issue.  You've used the long dash (–) instead of the short dash (-).  You can do a simple find replace of the long dash with the short dash to fix this.

File: i18n/de.json [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+    "dgc-learnmore-modal-step1": "<strong>Schritt 1:</strong> Hole dir eine Brieftasche.",
+    "dgc-learnmore-modal-step2": "<strong>Schritt 2:</strong> Hol dir etwas Dogecoin.",
+    "dgc-learnmore-modal-step3": "<strong>Schritt 3:</strong> Benutze dein Dogecoin.",
+    "dgc-learnmore-modal-step4": "<strong>Schritt 4:</strong> Auf dem neuesten Stand bleiben",
+    "dgc-learnmore-modal-more-info": "Mehr Informationen gibt es in der Anleitung weiter unten.",
+    "dgc-learnmore-modal-getting-started": "Anleitung zu den ersten Schritten",
+    "dgc-community-modal-title": "COMMUNITY:",
+    "dgc-community-modal-get-involved": "Engagiere dich mit Dogecoin!",
+    "dgc-community-modal-participate": "Nimm in unserer Reddit-Community teil, chatte auf unserem IRC (Internet-Chat)-Kanal, oder hab Spa&szlig; in unserem Forum.",
+    "dgc-community-modal-reddit-desc": " - Der Hauptteil der Dogecoin-Community.",
```
Comment (langerhans) [inline i18n/de.json:86]: That's not really true. Dogecoin doesn't revolve around reddit ;)

Comment (iw0nderhow) [inline i18n/de.json:86]: Yeah, it revolves around the moon :P

File: i18n/de.json [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+    "dgc-getting-started-desktop": "Desktop-Brieftasche",
+    "dgc-getting-started-phone": "Brieftasche auf dem Smartphone",
+    "dgc-getting-started-online": "Online-Brieftasche",
+    "dgc-getting-started-paper": "Brieftasche auf Papier",
+    "dgc-getting-started-choose": "W&auml;hle eine Art von Wallet aus, um zu beginnen.",
+    "dgc-getting-started-desktop-os": "W&auml;hle das Betriebssystem deines Computers aus.",
+    "dgc-getting-started-phone-os": "W&auml;hle das Betriebssystem deines Smartphones aus.",
+    "dgc-getting-started-online-choice": "W&auml;hle deine Online-Brieftasche aus.",
+    "dgc-getting-started-online-choice": "W&auml;hle deine Online-Brieftasche aus.",
+    "dgc-getting-started-online-choice": "W&auml;hle deine Online-Brieftasche aus.",
```
Comment (langerhans) [inline i18n/de.json]: Is that supposed to be here three times?

Comment (iw0nderhow) [inline i18n/de.json]: @langerhans It is there in every other translation file too, so I left it
there, but I don't think so.

File: i18n/de.json (hunk 3 above)
Comment (langerhans) [inline i18n/de.json:86]: No, that was serious, I even said that in my last interview.

Comment (iw0nderhow) [inline i18n/de.json:86]: @langerhans 
How should I put it then? "ein großer Teil" maybe? "ein wichtiger
Bestandteil"?

**Comment by iw0nderhow** [comment]
Posted this on [Reddit](http://www.reddit.com/r/dogecoingermany/comments/2k6wa4/deutsches_dogecoincom/)

**Comment by dhpollack** [comment]
@karottenbaum This is ready to be merged, right?  I just tested it on my phone and it looks good to go.

@PowerLemons assuming carrot tree has nothing to add, we should probably merge this.

**Comment by PowerLemons** [comment]
Looks good, thank you very much for contributing to the site! Merged.

**Comment by iw0nderhow** [comment]
Thanks!

**Pull request merged**